content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/cpp.md

```diff
@@ -23,153 +23,169 @@ further_reading:
   {
-    auto span_a = tracer->StartSpan("A");
-    span_a->SetTag("tag", 123);
-    auto span_b = tracer->StartSpan("B", {opentracing::ChildOf(&span_a->context())});
-    span_b->SetTag("tag", "value");
+    auto span_a = tracer.create_span();
+    span_a.set_name("A");
+    span_a.set_tag("tag", "123");
+    auto span_b = span_a.create_child();
+    span_b.set_name("B");
+    span_b.set_tag("tag", "value");
   }
 
-  tracer->Close();
   return 0;
 }
 ```
 
-Link against `libdd_opentracing` and `libopentracing`, making sure that they are both in your `LD_LIBRARY_PATH`:
-
 ```bash
-g++ -std=c++14 -o tracer_example tracer_example.cpp -ldd_opentracing -lopentracing
-./tracer_example
+cmake -B build .
+cmake --build build --target tracer_example -j
+
+./build/tracer_example
+DATADOG TRACER CONFIGURATION - {"collector":{"config":{"event_scheduler":{"type":"datadog::tracing::ThreadedEventScheduler" ... }}}
 ```
 
-### Dynamic loading
+{{% /tab %}}
+
+{{% tab "Manual" %}}
 
 ```bash
+# Requires the "jq" command, which can be installed via
+# the package manager:
+#   - APT: `apt install jq`
+#   - APK: `apk add jq`
+#   - YUM: `yum install jq`
+if ! command -v jq >/dev/null 2>&1; then
+  >&2 echo "jq command not found. Install using the local package manager."
+  exit 1
+fi
+
+# Gets the latest release version number from GitHub.
 get_latest_release() {
-  wget -qO- "https://api.github.com/repos/$1/releases/latest" |
-    grep '"tag_name":' |
-    sed -E 's/.*"([^"]+)".*/\1/';
+  curl --silent "https://api.github.com/repos/$1/releases/latest" | jq --raw-output .tag_name
 }
-DD_OPENTRACING_CPP_VERSION="$(get_latest_release DataDog/dd-opentracing-cpp)"
-OPENTRACING_VERSION="$(get_latest_release opentracing/opentracing-cpp)"
-# Download and install OpenTracing-cpp
-wget https://github.com/opentracing/opentracing-cpp/archive/${OPENTRACING_VERSION}.tar.gz -O opentracing-cpp.tar.gz
-mkdir -p opentracing-cpp/.build
-tar zxvf opentracing-cpp.tar.gz -C ./opentracing-cpp/ --strip-components=1
-cd opentracing-cpp/.build
-cmake ..
-make
-make install
-# Install dd-opentracing-cpp shared plugin.
-wget https://github.com/DataDog/dd-opentracing-cpp/releases/download/${DD_OPENTRACING_CPP_VERSION}/linux-amd64-libdd_opentracing_plugin.so.gz
-gunzip linux-amd64-libdd_opentracing_plugin.so.gz -c > /usr/local/lib/libdd_opentracing_plugin.so
-```
 
-Include `<opentracing/dynamic_load.h>` and load the tracer from `libdd_opentracing_plugin.so`:
+DD_TRACE_CPP_VERSION="$(get_latest_release DataDog/dd-trace-cpp)"
+
+# Download and install dd-trace-cpp library.
+wget https://github.com/DataDog/dd-trace-cpp/archive/${DD_TRACE_CPP_VERSION}.tar.gz -O dd-trace-cpp.tar.gz
+mkdir dd-trace-cpp && tar zxvf dd-trace-cpp.tar.gz -C ./dd-trace-cpp/ --strip-components=1
+cd dd-trace-cpp
+
+# Download and install the correct version of dd-trace-cpp.
+# Configure the project, build it, and install it.
+cmake -B build .
+cmake --build build -j
+cmake --install build
+```
 
```

Comment:
```suggestion
After installing the `dd-trace-cpp` library, you can use it in your C++ code to create spans and traces. Here's an example:
```
^ I think we need to add more introduction to these code examples so it's clear for users. Something like this (please check).